plugin/compile-scss.js

```diff
@@ -296,11 +274,7 @@ class SassCompiler extends MultiFileCachingCompiler {
   }
 }
 
-
 function _getRealImportPathFromIncludes(importPath, getRealImportPathFn){
-
-  _prepareNodeSassOptions();
-
   let possibleFilePath, foundFile;
```

Comment:
No longer load the configuration every time an attempt is made to load a file from node modules (or whatever else is specified in the configuration)